src/card-product.ts

```diff
@@ -0,0 +1,236 @@
+'use strict'
+
+import type {
+  Marqeta,
+  MarqetaOptions,
+} from './'
+
+export interface ExpirationOffset {
+  unit?: string;
+  value?: bigint;
+}
+
+export interface AddressType {
+  firstName?: string;
+  middleName?: string;
+  lastName?: string;
+  address1: string;
+  address2?: string;
+  city?: string;
+  state?: string;
+  zip?: string;
+  country?: string;
+  phone?: string;
+  postalCode?: string;
+}
+
+export interface AddressVerification {
+  avMessages?:{
+    validate?: boolean;
+    declineOnAddressNumberMismatch?: boolean;
+    declineOnPostalCodeMismatch?: boolean;
+  };
+  authMessages?: {
+    validate?: boolean;
+    declineOnAddressNumberMismatch?: boolean;
+    declineOnPostalCodeMismatch?: boolean;
+  };
+}
+
+export interface Shipping {
+  method?: string;
+  returnAddress?: AddressType;
+  recipientAddress?: AddressType;
+  careOfLine?: string;
+}
+
+export interface Text {
+  nameLine1: {
+    value: string;
+  };
+  nameLine2?: {
+    value: string;
+  };
+  nameLine3?: {
+    value: string;
+  };
+}
+
+export interface ImagesCarrier {
+  name?: string;
+  message1?: string;
+}
+
+export interface Images {
+  card?: {
+    name?: string;
+    thermalColor?: string;
+  };
+  carrier?: ImagesCarrier;
+  signature?: {
+    name: string;
+  };
+  carrierReturnWindow?: {
+    name: string;
+  };
+}
+
+export interface Carrier {
+  templateId?: string;
+  logoFile?: string;
+  logoThumbnailFile?: string;
+  messageFile?: string;
+  messageLine?: string;
+}
+
+export interface POI {
+  other: {
+    allow?: boolean
+    cardPresenceRequired?: boolean;
+    cardholderPresenceRequired?: boolean;
+    track1DiscretionaryData?: string;
+    track2DiscretionaryData?: string;
+  };
+  ecommerce?: boolean;
+  atm?: boolean;
+}
+
+export interface CustomerAuthentication {
+  scaContactlessTransactionLimit?: number;
+  scaContactlessCumulativeAmountLimit?: number;
+  scaContactlessTransactionsCountLimit?: bigint;
+  scaContactlessTransactionsCurrency?: string;
+  scaLvpTransactionLimit?: number;
+  scaLvpCumulativeAmountLimit?: number;
+  scaLvpTransactionsCountLimit?: bigint;
+  scaLvpTransactionsCurrency?: string;
+}
+
+export interface Controls {
+  acceptedCountriesToken?: string; //50 char max
```

Comment:
Minor point, but you have in other places put a space between the `//` and the rule... here you don't. I like your use of the space... it's easier to read.